WRITERS-NOTES.md

```diff
@@ -0,0 +1,232 @@
+| <i class="uil uil-shield"></i>               | Shield                             | PMM UI - Server admin                |
+| <i class="uil uil-star"></i>                 | Star                               | PMM UI - Dashboard favourites        |
+| <i class="uil uil-sync"></i>                 | Twin backward arrows               | PMM UI - Refresh dashboard           |
+| <i class="uil uil-thumbs-down"></i>          | Hand, thumbs down                  | For Benefits/Drawbacks tables        |
+| <i class="uil uil-thumbs-up"></i>            | Hand, thumbs up                    | For Benefits/Drawbacks tables        |
+| <i class="uil uil-times"></i>                | Large 'X'                          | PMM UI                               |
+| <i class="uil uil-toggle-off"></i>           | Toggle (off)                       | PMM UI - Toggle switch               |
+| <i class="uil uil-toggle-on"></i>            | Toggle (on)                        | PMM UI - Toggle switch               |
+| <i class="uil uil-trash-alt"></i>            | Trash can                          | PMM UI - Various 'Delete' operation  |
+
+Custom (in-house design) icons are defined as SVG code in `variables.yml`.
+
+| Usage                  | Description                          | Used where                       |
+| ---------------------- | ------------------------------------ | -------------------------------- |
+| `{{icon.addinstance}}` | Thin + in circle                     | PMM Add Instance                 |
+| `{{icon.checks}}`      | Checkbox list items                  | Inventory list                   |
+| `{{icon.inventory}}`   | Checkbox list items                  | PMM Inventory & Inventory List   |
+| `{{icon.dashboards}}`  | Abstract blocks assembly             | PMM Dashboards                   |
+| `{{icon.node}}`        | Circle surrounded by smaller circles | Node dashboards                  |
+| `{{icon.percona}}`     | Percona logo                         | Wherever PMM home icon is needed |
+
+## Symbols
+
+While MkDocs will automatically replace certain strings with symbols, it's preferable where possible to use unicode symbols for other icons, so that they appear when the raw Markdown is exported as HTML and imported into Google Docs.
+
+| For | Use |
+| --- | --- |
+| --> | →   |
+
+## Language
+
+We have attempted to eschew traditional terminology used in software manuals. Some examples:
+
+- "Setting up" instead of "installation and configuration"
+- "Before you start" instead of "Prerequisites"
+
+There are no "introduction" or "overview" sections. These texts are just there under the title.
+
+Section titles are deliberately short. For example, in Setting up/Server/Docker, the 'Run' section shows how to run the docker image for PMM Server. The docs are for 'PMM', the section is 'Server' and subsection 'Docker'. That's what we're running.
```

Comment:
I think we should add such sections only after beta, once we have enough topics we can reference